### DISCUSSION TITLE
Is it possible to use different abbreviation than IFC ?

### DISCUSSION THREAD

**Discussion by tapika** [microsoft/ifc#72] [General]
IFC typically associated with "Industry Foundation Classes": In the field of building information modeling (BIM), IFC is a standard for data exchange that enables different software applications to share building and construction data.

If for example I want to find out whether IFC SDK is used with godot - I type in google "godot" "IFC", but it gives me plenty unrelated stuff about which I don't care.

Would it be possible to rename IFC to something else, more descriptive or use different abbreviation ?

**Comment by gdr-at-ms** [comment]
"IFC" has been used for over a decade in the MSVC toolset, C++ Modules educational materials, etc.

You can try "Microsoft IFC SDK" if you're looking for the Microsoft SDK for the IFC Specification.
